services/ui-auth/handlers/createUsers.js

```diff
@@ -12,7 +12,6 @@ async function myHandler(event, context, callback) {
       UserPoolId: userPoolId,
       Username: users[i].username,
       DesiredDeliveryMediums: ["EMAIL"],
-      TemporaryPassword: "Abc@321456!",
```

Comment:
Nope, the docs say it's optional.  If it's not specified, Amazon will generate a temp password for the user.
Since we immediately set the permanent password for the user from an SSM value, we don't care about the temp pw.